config/runtime.exs

```diff
@@ -78,7 +78,7 @@ end
 
 # Optional
 sentry_dsn = System.get_env("SENTRY_DSN")
-mailgun_api_key = System.get_env("MAILGUN_API_KEY")
+mailer_adapter = System.get_env("MAILER_ADAPTER", "Swoosh.Adapters.Mailgun")
```

Comment:
Use the Mailgun as a default to make this change backwards compatible. 

I'm thinking if we could use the Local adapter as a default https://hexdocs.pm/swoosh/Swoosh.Adapters.Local.html. That way we dont need a try/catch when calling `deliver` when running locally.

Reply:
The nice about the local adapter is that you can add a route when running dev to view the emails that are sent https://hexdocs.pm/swoosh/Plug.Swoosh.MailboxPreview.html

Reply:
@fmterrorf i like this idea! would you mind adding it to this PR, or would you prefer handling in a separate PR?

Reply:
I can add it to this PR!

Reply:
@reichert621 here's a screenshot of the mailbox preview 
![image](https://user-images.githubusercontent.com/77761194/118555747-3fabeb00-b720-11eb-8422-d686a11ffa51.png)